src/utils/provider.ts

```diff
@@ -55,24 +56,35 @@ export function addAuthorize(nuxt, strategy) {
           return next()
         }
 
+        let data: object|string = {
+          client_id: clientID,
+          client_secret: clientSecret,
+          refresh_token: refreshToken,
+          grant_type: grantType,
+          response_type: responseType,
+          redirect_uri: redirectUri,
+          audience,
+          code_verifier: codeVerifier,
+          code
+        }
+
+        let headers = {
+          Accept: 'application/json',
+          'Content-Type': 'application/json'
+        }
+
+        if (useForms) {
+          delete data['code_verifier']
```

Comment:
Why delete code_verifier? Won't that make it less secure?

Reply:
You're right, works without deleting that field, I've removed this line.